prowler/lib/outputs/html.py

```diff
@@ -194,6 +200,7 @@ def add_html_header(file_descriptor, audit_info):
                     <th scope="col">Risk</th>
                     <th scope="col">Recomendation</th>
                     <th style="width:5%" scope="col">Recomendation URL</th>
+                    <th scope="col">Compliance</th>
```

Comment:
We need to resize the "Resource Tags" column, is bigger than the title or the resource extended.

Reply:
Done